tracer/test/Datadog.Trace.OpenTracing.IntegrationTests/OpenTracingSendTracesToAgent.cs

```diff
@@ -4,99 +4,122 @@
+                                                  .Start();
+                span.Finish();
+
+                var spans = agent.WaitForSpans(1);
+
+                var receivedSpan = spans.Should().ContainSingle().Subject;
+                CompareSpans(receivedSpan, span);
+                // receivedSpan.SpanId.Should().Be(span.DDSpan.SpanId);
+                // receivedSpan.TraceId.Should().Be(span.DDSpan.TraceId);
+                // receivedSpan.Resource.Should().Be(span.DDSpan.ResourceName);
+                // receivedSpan.Name.Should().Be(span.DDSpan.OperationName);
+            }
         }
 
-        [Fact(Skip = "Run manually")]
-        public async void Utf8Everywhere()
+        [Fact]
+        public void Utf8Everywhere()
         {
-            var span = (OpenTracingSpan)_tracer.BuildSpan("Aᛗᚪᚾᚾᚪ")
-                                               .WithTag(DatadogTags.ResourceName, "η γλώσσα μου έδωσαν ελληνική")
-                                               .WithTag(DatadogTags.ServiceName, "На берегу пустынных волн")
-                                               .WithTag("யாமறிந்த", "ნუთუ კვლა")
-                                               .Start();
-            span.Finish();
-
-            // Check that the HTTP calls went as expected
-            await _httpRecorder.WaitForCompletion(1);
-            Assert.Single(_httpRecorder.Requests);
-            Assert.Single(_httpRecorder.Responses);
-            Assert.All(_httpRecorder.Responses, (x) => Assert.Equal(HttpStatusCode.OK, x.StatusCode));
-
-            var trace = _httpRecorder.Traces.Single();
-            MsgPackHelpers.AssertSpanEqual(span.DDSpan, trace.Single());
+            int agentPort = TcpPortProvider.GetOpenPort();
+
+            using (var agent = new MockTracerAgent(agentPort))
+            {
+                var settings = new TracerSettings
+                {
+                    AgentUri = new Uri($"http://127.0.0.1:{agent.Port}"),
+                    TracerMetricsEnabled = false,
+                };
+
+                var innerTracer = new Tracer(settings, agentWriter: null, sampler: null, scopeManager: null, statsd: null);
+                var tracer = new OpenTracingTracer(innerTracer);
+
+                var span = (OpenTracingSpan)tracer.BuildSpan("Aᛗᚪᚾᚾᚪ")
+                                                  .WithTag(DatadogTags.ResourceName, "η γλώσσα μου έδωσαν ελληνική")
+                                                  .WithTag(DatadogTags.ServiceName, "На берегу пустынных волн")
+                                                  .WithTag("யாமறிந்த", "ნუთუ კვლა")
+                                                  .Start();
+                span.Finish();
+
+                var spans = agent.WaitForSpans(1);
+
+                var receivedSpan = spans.Should().ContainSingle().Subject;
+                CompareSpans(receivedSpan, span);
+                // receivedSpan.SpanId.Should().Be(span.DDSpan.SpanId);
```

Comment:
Can these lines be deleted now?

Reply:
yep, done in https://github.com/DataDog/dd-trace-dotnet/pull/1980/commits/70c7d24aaeb80d1c817c9ba31d8e0393b2135a70